documentation/src/docs/security/pravega-security-configurations.md

```diff
@@ -9,46 +9,65 @@ You may obtain a copy of the License at
-|singlenode.passwd|If _auth_ is enabled, this represents the password used for internal communication between segment store and controller|Empty|
+This document describes the security configuration parameters for Pravega, in both distributed and standalone modes.
+
+## Security Configuration Parameters in Distributed Mode
+
+In the distributed mode, Controllers and Segment Stores are configured individually. The following sub-sections describe
+their Transport Layer Security (TLS) and Auth (Authentication and Authorization) parameters.
+
+
+### Segment store <a name = "pravega-segment-store"></a>
+
+|Parameter|Description|Default Value|Feature|
+|---------|-------|-------------|------------|
+| pravegaservice.enableTls | Whether to enable TLS for client-server communications. | False | TLS |
+| pravegaservice.certFile | Path of the X.509 PEM-encoded server certificate file for the service. | Empty | TLS |
+| pravegaservice.keyFile | Path of the PEM-encoded private key file for the service. | Empty | TLS |
+| pravegaservice.secureZK | Whether to enable TLS for communication with Apache Zookeeper. | False | TLS |
+| pravegaservice.zkTrustStore | Path of the truststore file in `.jks` format for TLS connections with Apache Zookeeer. | Empty | TLS |
+| pravegaservice.zkTrustStorePasswordPath | Path of the file containing the password of the truststore used for TLS connections with Apache Zookeeper. | Empty | TLS |
+| autoScale.tlsEnabled | Whether to enable TLS for internal communication with the Controllers. | False | TLS |
+| autoScale.tlsCertFile | Path of the PEM-encoded X.509 certificate file used for TLS connections with the Controllers. | Empty | TLS |
+| autoScale.validateHostName | Whether to enable hostname verification for TLS connections with the Controllers. | True | TLS |
+| autoScale.authEnabled | Whether to enable authentication and authorization for internal communications with the Controllers. | False | Auth |
+| autoScale.tokenSigningKey | The key used for signing the delegation tokens. | Empty | Auth |
+| bookkeeper.tlsEnabled | Whether to enable TLS for communication with Apache Bookkeeper. | False | TLS |
+| bookkeeper.tlsTrustStorePath | Path of the truststore file in `.jks` format for TLS connections with Apache Bookkeeper. | Empty | TLS |
+
+
+### Controller <a name ="pravega-controller"></a>
+
+|Parameter|Details|Default Value|Feature|
+|---------|-------|-------------|-------|
+| controller.auth.tlsEnabled | Whether to enable TLS for client-server communication. | False | TLS |
+| controller.auth.tlsCertFile | Path of the X.509 PEM-encoded server certificate file for the service. | Empty | TLS |
+| controller.auth.tlsKeyFile | Path of the PEM-encoded private key file for the service. | Empty | TLS |
+| controller.auth.tlsTrustStore | Path of the PEM-encoded truststore file for TLS connections with Segment Stores. | Empty | TLS |
```

Comment:
PEM format

Reply:
As described elsewhere, the existing phrase is better suited in this context.